node/src/neighborhood/gossip_producer.rs

```diff
@@ -335,7 +335,8 @@ mod tests {
     }
 
     #[test]
-    fn produce_debut_creates_a_gossip_to_a_target_about_ourselves_when_accepting_connections() {
+    fn produce_debut_or_ipchange_creates_a_gossip_to_a_target_about_ourselves_when_accepting_connections(
```

Comment:
Maybe this is my weak English but I cannot say for sure just from the name of the test whether it is the source or the target who is accepting connections. 


I could read it from the code or even deduce it but the name would be best if it was enough to make it clear.

Reply:
Rearranged